src/fairdrive-protocol.ts

```diff
@@ -39,46 +46,78 @@ export class FairdriveProtocol {
       this.users[username] = address
     } else if (mnemonic) {
       try {
-        this.users[username] = Wallet.fromMnemonic(mnemonic).address
+        const wallet = Wallet.fromMnemonic(mnemonic)
+        this.users[username] = wallet.address
```

Comment:
why not save the `wallet` itself?

Reply:
The principle is taken from the original code. As far as I understand, this is necessary to manage multiple accounts. Not sure if we need support for multiple accounts. On the client side, it is most likely not needed. On the server side, it is needed to fully mimic the behaviour of the current version of FairOS. 
But is it our way? 
This needs to be discussed.